src/__experimental__/components/textbox/__snapshots__/textbox.spec.js.snap

```diff
@@ -1,54 +1,22 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Textbox renders with InputPresentation and Input and all props passed to Input 1`] = `
```

Comment:
Either the snapshot is incorrect or the description needs changing here, I think it might be the snapshot but needs investigating why

Reply:
InputPresentation and Input are part of Textbox. I am making an educated guess that is what you meant? (I mean then, description is ok)

Reply:
oh and btw, test-update seems to have removed InputPresentation and Input from snapshot automatically.

Reply:
ok, updated.